dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/container/internal/SipContainerComponent.java

```diff
@@ -116,22 +117,24 @@ public static ComponentContext getContext() {
 	 *            populated/provided by config admin
 	 */
 	protected void activate(ComponentContext context, Map<String, Object> properties) {
-		if (c_logger.isTraceDebugEnabled())
-			c_logger.traceDebug("SipContainerComponent activated", properties);
-		m_context = context;
-		try {
-			PropertiesStore.getInstance().getProperties().updateProperties(properties);
-			initContainer();
-		} catch (ParserConfigurationException e) {
-			if (c_logger.isErrorEnabled())
-				c_logger.error("error.initialize.sip.container");
-		}
-        useNetty = MetatypeUtils.parseBoolean(
-                "sipContainer", USE_NETTY,
-                properties.get(USE_NETTY), true);
-
-		genericEndpointRef.activate(context);
-		sipMessageFactoryRef.activate(context);
+           if (c_logger.isTraceDebugEnabled())
+               c_logger.traceDebug("SipContainerComponent activated", properties);
+           m_context = context;
+           try {
+              PropertiesStore.getInstance().getProperties().updateProperties(properties);
+              initContainer();
+           } catch (ParserConfigurationException e) {
+              if (c_logger.isErrorEnabled())
+                 c_logger.error("error.initialize.sip.container");
+           }
+           useNetty = (!ProductInfo.getBetaEdition())?false:MetatypeUtils.parseBoolean(
+              "sipContainer", USE_NETTY,
+              properties.get(USE_NETTY), true);
+           if (c_logger.isTraceDebugEnabled() && useNetty)
+              c_logger.traceDebug("SipContainerComponent activate: useNetty=true");
+
+           genericEndpointRef.activate(context);
+           sipMessageFactoryRef.activate(context);
```

Comment:
I think every line shows changed b/c of formatting differences tabs vs spaces etc. If we can clean that up it would be nice so we can more easily see what's changed at a glance.